charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.36.0
+version: 0.37.0
```

Comment:
I would say we should do a patch version bump in this case

Reply:
done